pkg/version/version.go

```diff
@@ -1,4 +1,57 @@
 package version
 
+import (
+	"fmt"
+	"regexp"
+
+	version "github.com/hashicorp/go-version"
+	coreops "github.com/portworx/sched-ops/k8s/core"
+)
+
 // Version will be overridden with the current version at build time using the -X linker flag
 var Version string
+
+var (
+	kbVerRegex = regexp.MustCompile(`^(v\d+\.\d+\.\d+)(.*)`)
+)
+
+// RegisterV1CRD returns true of k8s version is greater than 1.16
+func RegisterV1CRD() (bool, error) {
+	k8sVersion, _, err := GetFullVersion()
+	if err != nil {
+		return false, err
+	}
+	k8sVer1_16, err := version.NewVersion("1.16")
```

Comment:
since we already have integration job around for stork 2.6.4 with k8s 1.21, should we up this version to `1.21`. It will reduce upgrade test scope

Reply:
Regardless of the test matrix, we need to support stork deployments in k8s 1.16, so we still need this.

Reply:
this check is to decide whether to register v1 or v1beta crds version? v1betav1 version is supported upto k8s version v1.21